lib/beacon/content.ex

```diff
@@ -19,6 +21,7 @@ defmodule Beacon.Content do
   alias Beacon.Repo
   alias Beacon.Types.Site
   alias Ecto.Changeset
+  import Ecto.Query
```

Comment:
Ideally we want to keep module directives like this organized in blocks of descending impact, with an empty line in-between. i.e.

```elixir
use FooModule

import BarModule

alias BazModule
alias BongModule

require SomethingElse
```

Since `import` has a greater impact on this module than `alias`, it should go above